project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 // Use the Play sbt plugin for Play projects
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.9.5")
+addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.5")
```

Comment:
See:

* https://github.com/guardian/maintaining-scala-projects/issues/4

Note that the Play `sbt-plugin` is only required for the `exampleApp` app, not any of the other subprojects.